db/seeds.rb

```diff
@@ -24,5 +24,13 @@
   else
     flag = false
   end
-  patient.pregnancies.create({last_menstrual_period_time: DateTime.new(2016,1,1), urgent_flag: flag})
+  pregnancy = patient.pregnancies.create({last_menstrual_period_time: DateTime.new(2016,1,1), urgent_flag: flag})
+  8.times do
+    pregnancy.calls.create({status: 'Left voicemail', creating_user_id: user.id})
+  end
+  if(patient.name[-1, 1].to_i == 0) then
```

Comment:
Totally agree, this version was the result of some last minute copy paste.

I tried to follow your suggestion about separating it from the block like this:
```ruby
10.times do
  Patient.find(name: 'Patient 0').pregnancy.calls.create({status: 'Reached patient', creating_user_id: user.id})
end
```
But I don't think that works because 1 patient can have multiple pregnancies.

So I used the `if patient.name == 'Patient 0'` approach and that seems to work. Let me know if you think that looks good from a readability standpoint or if there is any better alternative.